src/unit/test_files.h

```diff
@@ -240,6 +241,7 @@ unitTest __test_kvstore_c[] = {{"test_kvstoreAdd16Keys", test_kvstoreAdd16Keys},
 unitTest __test_networking_c[] = {{"test_writeToReplica", test_writeToReplica}, {"test_postWriteToReplica", test_postWriteToReplica}, {"test_backupAndUpdateClientArgv", test_backupAndUpdateClientArgv}, {"test_rewriteClientCommandArgument", test_rewriteClientCommandArgument}, {NULL, NULL}};
 unitTest __test_object_c[] = {{"test_object_with_key", test_object_with_key}, {NULL, NULL}};
+unitTest __test_bitops_c[] = {{"test_popcount", test_popcount}, {NULL, NULL}};
```

Comment:
Seems this file isn't updated. Build the unit tests locally to update this file and then commit it.